modules/creating-runtimeclass.adoc

```diff
@@ -18,7 +18,7 @@ apiVersion: node.k8s.io/v1beta1
 kind: RuntimeClass
 metadata:
   name: <runtime_class_name> <1>
-handler: 'docker'
+handler: 'containerd'
```

Comment:
@mburke5678 the handler should be handler: 'runhcs-wcow-process'
Here is the [runtime class kubernetes](https://kubernetes.io/docs/concepts/containers/runtime-class/#hahahugoshortcode-s3-hbhb) doc on it.
The value for handler in the above yaml comes from the default [containerd config](https://github.com/openshift/windows-machine-config-operator/blob/de1b0d1c500388da592f54d63b3a6ff31f5d99a9/pkg/internal/containerd_conf.toml#L93) file we use.
Please let me know if you need any more pointers for this.